utils/stall_free.hh

```diff
@@ -56,5 +56,15 @@ std::list<T> stall_free_merge(std::list<T>& list1, const std::list<T>& list2, Co
     return list_output;
 }
 
+
+// Clear items in a list without stall. Must run inside a seastar thread.
+template<class T>
+void stall_free_clear(std::list<T>& list) {
```

Comment:
We use the suffix "_gently" in the cache for such things. Let's be consistent and call it merge_gently, clear_gently.

Reply:
OK.